modules/mo_pytorch/mo_extensions/front/pytorch/common.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+
+def get_pads(padding):
```

Comment:
Will it works for other attributes such strides, kernel_size, dilations?

Reply:
tuple

Reply:
No, it won't. We have doubled dimension for each value of padding.